framework/src/Volo.Abp.UI.Navigation/Volo.Abp.UI.Navigation.csproj

```diff
@@ -23,4 +23,11 @@
     <ProjectReference Include="..\Volo.Abp.UI\Volo.Abp.UI.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Fody" Version="6.1.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
```

Comment:
We can remove these because it is already configured in `configureawait.props`.

Reply:
Resolved apologies